src/frontend/pages/Notes/Labels/Labels.jsx

```diff
@@ -2,18 +2,20 @@ import React from "react";
 import { ToastContainer } from "react-toastify";
 import { useArchive } from "../../../contexts/archive-context";
 import { useEdit } from "../../../contexts/edit-context";
+import { useFilter } from "../../../contexts/filters-context";
 import { useNote } from "../../../contexts/note-context";
```

Comment:
create an index file and use barrel import

Reply:
yeah will do it in refactor branch